docs/index.md

```diff
@@ -93,25 +93,25 @@ Hundreds of organisations and packages are using *pydantic*, including:
   [for subprojects](https://github.com/samuelcolvin/pydantic/issues/773).
 
 **Microsoft**
-: are using *pydantic* (via FastAPI) for 
-  [numerous services](https://github.com/tiangolo/fastapi/pull/26#issuecomment-463768795) some of which are 
+: is using *pydantic* (via FastAPI) for 
```

Comment:
it's debatable, bit I think "are" reads better here.

Reply:
Looks like this is a bit of an Anglicism -- https://english.stackexchange.com/questions/133105/organisation-singular-or-plural/133106#133106. Happy to keep it that way for the euro flair -- I'll rework them all to be consistent later.